capstone/src/main/java/com/google/univiz/CollegeDataApi.java

```diff
@@ -0,0 +1,13 @@
+package com.google.univiz;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * CollegeDataApi is an interface for the CollegeDataApiImpl class that interacts with external APIs
+ * to send REST requests.
```

Comment:
I would encourage leaving out implementation details in interface javadoc. In particular, the name of the implementing class (which could change) and the means by which the implementation will fetch data (REST requests). Such details can quickly become stale.

This javadoc could be simply:

/** Provides access to college related data. */